doc/design/proposal-chaining-middleware.md

```diff
@@ -263,3 +263,42 @@ export async function load(
 }
 ```
 </details>
+
+## Chaining `readFile` hooks
+
+Say you had a chain of three loaders:
+
+* `zip` adds a virtual filesystem layer for in-zip access
+* `tgz` does the same but for tgz archives
+* `warc` does the same for warc archives.
+
+Following the pattern of `--require`:
+
+```console
+node \
+  --loader zip \
+  --loader tgz \
+  --loader warc
+```
+
+These would be called in the following sequence: `zip` calls `tgz`, which calls `warc`. Or in JavaScript terms, `zip(tgz(warc(input)))`:
+
+Load hooks would have the following signature:
+
+```ts
+export async function readFile(
+  url: string,               // A URL that point to a location; whether the file
```

Comment:
```suggestion
  url: string,               // A URL pointing to a location; whether the file
```

Should be either "that points" or "pointing" ("pointing" works better for line-length)